src/app/components/Figure/index.jsx

```diff
@@ -1,50 +1,8 @@
-import React from 'react';
-import { string } from 'prop-types';
 import styled from 'styled-components';
-import Caption from './Caption';
-import VisuallyHiddenText from '../VisuallyHiddenText';
 import { GEL_SPACING_DBL } from '../../lib/constants/styles';
 
-const copyrightOffscreenText = 'Copyright';
-
-const renderCopyright = copyright => {
-  const copyrightText = `${copyrightOffscreenText} ${copyright}`;
-  return copyright !== 'BBC' ? (
-    <VisuallyHiddenText>{copyrightText}</VisuallyHiddenText>
-  ) : null;
-};
-
-const renderCaption = caption =>
-  caption ? <Caption>{caption}</Caption> : null;
-
-const StyledFigure = styled.figure`
+export default styled.figure`
```

Comment:
We shouldn't default export react components as it is bad for debugging (in react dev tools) because they are exported as unnamed functions.

Reply:
Sorted